docs/components/plugins/api/APISectionMethods.tsx

```diff
@@ -47,51 +49,65 @@ export const renderMethod = (
   const signatures = method.signatures || (method as PropData)?.type?.declaration?.signatures || [];
   const HeaderComponent = exposeInSidebar ? H3Code : H4Code;
   return signatures.map(
-    ({ name, parameters, comment, type }: MethodSignatureData | TypeSignaturesData) => (
-      <div
-        key={`method-signature-${method.name || name}-${parameters?.length || 0}`}
-        css={[STYLES_APIBOX, !exposeInSidebar && STYLES_APIBOX_NESTED]}>
-        <APISectionDeprecationNote comment={comment} />
-        <APISectionPlatformTags comment={comment} prefix="Only for:" />
-        <HeaderComponent tags={getTagNamesList(comment)}>
-          <InlineCode css={!exposeInSidebar ? STYLES_NOT_EXPOSED_HEADER : undefined}>
-            {apiName && `${apiName}.`}
-            {`${method.name || name}(${parameters ? listParams(parameters) : ''})`}
-          </InlineCode>
-        </HeaderComponent>
-        {parameters && renderParams(parameters)}
-        <CommentTextBlock comment={comment} includePlatforms={false} />
-        {resolveTypeName(type) !== 'undefined' && (
-          <>
-            <div css={STYLES_NESTED_SECTION_HEADER}>
-              <H4>Returns</H4>
-            </div>
-            <UL hideBullets>
-              <LI>
-                <UndoIcon
-                  color={theme.icon.secondary}
-                  size={iconSize.small}
-                  css={returnIconStyles}
-                />
-                <APIDataType typeDefinition={type} />
-              </LI>
-            </UL>
-            {comment?.returns && (
-              <ReactMarkdown components={mdComponents}>{comment.returns}</ReactMarkdown>
-            )}
-          </>
-        )}
-      </div>
-    )
+    ({ name, parameters, comment, type }: MethodSignatureData | TypeSignaturesData) => {
+      const isProperty = method.kind === TypeDocKind.Property && !parameters?.length;
+
+      let methodName = (apiName && `${apiName}.`) ?? '';
+      methodName = methodName + (method.name || name);
+      if (!isProperty) {
+        methodName = `${methodName}(${parameters ? listParams(parameters) : ''})`;
+      }
```

Comment:
```suggestion
      let methodName = (apiName && `${apiName}.`) ?? '';
      methodName = methodName + (method.name || name);
      if (!isProperty) {
        methodName = `${methodName}(${parameters ? listParams(parameters) : ''})`;
      }
```

Can we extract this logic to the separate method like `getMethodName` and avoid `let`?  Also I feel like this construction is a bit less readable than separated `apiName` and `methodName`.